task/06-conditions-n-loops-tasks.js

```diff
@@ -293,10 +325,24 @@ function getDigitalRoot(num) {
  *   '[[][][[]]]' => true
  *   '[[][]][' => false
  *   '{)' = false
- *   '{[(<{[]}>)]}' = true 
+ *   '{[(<{[]}>)]}' = true
  */
 function isBracketsBalanced(str) {
-    throw new Error('Not implemented');
+    const closeBrackets = ["[", "(", "{", "<"];
+    const openBrackets = ["]", ")", "}", ">"];
+
+    let stack = [];
+
+    for (let i = 0; i < str.length; i++){
+        if (~closeBrackets.indexOf(str[i]))
+            stack.push(str[i]);
+        else if (openBrackets.indexOf(str[i]) !== closeBrackets.indexOf(stack.pop()))
+            return false;
+    }
+
+    if (stack.length > 0)
+        return false;
+    return true;
```

Comment:
We can just `return !(stack.length > 0);` 
Please, see https://javascript.info/logical-operators